doc/whats_new/v0.20.rst

```diff
@@ -85,6 +85,11 @@ Classifiers and regressors
   identical X values.
   :issue:`9432` by :user:`Dallas Card <dallascard>`
 
+- Fixed a bug in :class:`linear_model.LogisticRegression` where when using the
+  parameter ``multi_class='multinomial'``, the ``predict_proba`` method was
+  returning incorrect probabilities in the case of binary outcomes.
+  :issue:`9889`. By user `rwolst`.
```

Comment:
Please use the syntax: 
```
:issue:`9889` by :user:`rwolst`.
```

Reply:
Please use this issue number, not the original issue. Easier to check logs are complete

Reply:
Is that from your 0.19.1 experience? I would say that, as a random user, the original issue (rather than the PR) is generally a better source of understanding what the problem was, so I would be mildly in favor of using the original issue in the whats_new entry. The PR is better to understand how the problem was fixed.

Reply:
No, for something like the 0.19.0 release I tried to check that the changeloga included everything merged that wasn't docs, CI, etc. That is easier with pr#s here. If GitHub had an API to get the PRs closing an issue for a particular issue this would not be difficult...